### PR TITLE
PRODSECBUG-2347 Release Note Update

### PR DESCRIPTION
**Pull request by jfrontain**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will add a brief note that describes the PayPal Payflow Pro active carding activity update and provide a link to the informative Knowledge Base article. 

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.1OpenSource.html
- https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.html
- https://devdocs.magento.com/guides/v2.2/release-notes/ReleaseNotes2.2.8CE.html
- https://devdocs.magento.com/guides/v2.2/release-notes/ReleaseNotes2.2.8EE.html
- https://devdocs.magento.com/guides/v2.1/release-notes/ReleaseNotes2.1.17CE.html
- https://devdocs.magento.com/guides/v2.1/release-notes/ReleaseNotes2.1.17EE.html

whatsnew
Added information on Composer package updates to address PayPal Payflow Pro carding activity issues to the Release Notes for 2.3.1 [Open Source](https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.1OpenSource.html)/[Commerce](https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.html), 2.2.8 [Open Source](https://devdocs.magento.com/guides/v2.2/release-notes/ReleaseNotes2.2.8CE.html)/[Commerce](https://devdocs.magento.com/guides/v2.2/release-notes/ReleaseNotes2.2.8EE.html), and 2.1.17 [Open Source](https://devdocs.magento.com/guides/v2.1/release-notes/ReleaseNotes2.1.17CE.html)/[Commerce](https://devdocs.magento.com/guides/v2.1/release-notes/ReleaseNotes2.1.17EE.html).

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4103) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4103) before we can accept your contribution.<br/><hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ Follow these steps to download and apply this patch:
 6. Refresh the cache from the Admin (**System** > **Cache Management**).
 
 
+## PayPal Payflow Pro active carding activity update
+
+The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages  that update the CAPTCHA and Google reCAPTCHA extensions.  These packages  also include new options to run those checks on the PayPal Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all merchants download and install these packages for the security of their storefronts**.
```
Comment (lorikrell) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md]: The changes are on the form itself to use reCAPTCHA/CAPTCHA. They are not updating the modules. Suggested edit:

```suggestion
The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages that add an option for Google reCAPTCHA and CAPTCHA to the Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all Payflow Pro merchants download and install these packages for the security of their storefronts**.
```

File: guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md [hunk 2]
```diff
@@ -32,6 +32,13 @@ Follow these steps to download and apply this patch:
 6. Refresh the cache from the Admin (**System** > **Cache Management**).
 
 
+## PayPal Payflow Pro active carding activity update
+
+The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages  that update the CAPTCHA and Google reCAPTCHA extensions.  These packages  also include new options to run those checks on the PayPal Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all merchants download and install these packages for the security of their storefronts**.
```
Comment (lorikrell) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md]: Same as previous edit suggestion:

```suggestion
The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages that add an option for Google reCAPTCHA and CAPTCHA to the Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all Payflow Pro merchants download and install these packages for the security of their storefronts**.
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.8CE.md [hunk 3]
```diff
@@ -31,6 +31,16 @@ Follow these steps to download and apply this patch:
 
 6. Refresh the cache from the Admin (**System** > **Cache Management**).
 
+
+
+## PayPal Payflow Pro active carding activity update
+
+The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages  that update the CAPTCHA and Google reCAPTCHA extensions.  These packages  also include new options to run those checks on the PayPal Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all merchants download and install these packages for the security of their storefronts**.
```
Comment (lorikrell) [inline guides/v2.2/release-notes/ReleaseNotes2.2.8CE.md]: Same as previous edit suggestion:

```suggestion
The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages that add an option for Google reCAPTCHA and CAPTCHA to the Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all Payflow Pro merchants download and install these packages for the security of their storefronts**.
```

File: guides/v2.2/release-notes/ReleaseNotes2.2.8EE.md [hunk 4]
```diff
@@ -36,6 +36,12 @@ Follow these steps to download and apply this patch:
 6. Refresh the cache from the Admin (**System** > **Cache Management**).
 
 
+## PayPal Payflow Pro active carding activity update
+
+The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages  that update the CAPTCHA and Google reCAPTCHA extensions.  These packages  also include new options to run those checks on the PayPal Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all merchants download and install these packages for the security of their storefronts**.
```
Comment (lorikrell) [inline guides/v2.2/release-notes/ReleaseNotes2.2.8EE.md]: Same as previous edit suggestion:

```suggestion
The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages that add an option for Google reCAPTCHA and CAPTCHA to the Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all Payflow Pro merchants download and install these packages for the security of their storefronts**.
```

File: guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md [hunk 5]
```diff
@@ -29,6 +29,12 @@ Follow these steps to download and apply this patch:
 
 6. Refresh the cache from the Admin (**System** > **Cache Management**).
 
+## PayPal Payflow Pro active carding activity update
+
+The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages  that update the CAPTCHA and Google reCAPTCHA extensions.  These packages  also include new options to run those checks on the PayPal Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all merchants download and install these packages for the security of their storefronts**.
```
Comment (lorikrell) [inline guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md]: Same as previous edit suggestion:

```suggestion
The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages that add an option for Google reCAPTCHA and CAPTCHA to the Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all Payflow Pro merchants download and install these packages for the security of their storefronts**.
```

File: guides/v2.3/release-notes/ReleaseNotes2.3.1OpenSource.md [hunk 6]
```diff
@@ -27,6 +27,13 @@ Follow these steps to download and apply this patch:
 6. Refresh the cache from the Admin (**System** > **Cache Management**).
 
 
+## PayPal Payflow Pro active carding activity update
+
+The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages  that update the CAPTCHA and Google reCAPTCHA extensions.  These packages  also include new options to run those checks on the PayPal Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all merchants download and install these packages for the security of their storefronts**.
```
Comment (lorikrell) [inline guides/v2.3/release-notes/ReleaseNotes2.3.1OpenSource.md]: Same as previous edit suggestion:

```suggestion
The PayPal Payflow Pro integration in Magento is being actively targeted by carding activity. Magento has resolved these carding activity issues by providing Composer packages that add an option for Google reCAPTCHA and CAPTCHA to the Payflow Pro checkout form. See [PayPal Payflow Pro active carding activity](https://support.magento.com/hc/en-us/articles/360025515991)  for a full discussion of this issue and instructions on downloading these packages. **We strongly recommend that all Payflow Pro merchants download and install these packages for the security of their storefronts**.
```

**Comment by lorikrell** [comment]
running tests

**Comment by dshevtsov** [comment]
@lorikrell I suggest to label this update as Major update, as it adds a new release note

**Comment by lorikrell** [comment]
Thanks Dima! Working to get this merged today.

**Comment by lorikrell** [comment]
running tests

**Pull request merged**